CameraOverlay/mqtt_pub_test.py

```diff
@@ -0,0 +1,73 @@
+import sys

```

Comment:
Can you remove this file from the repo? It will now live in the MHP_DAS repo. We could potentially have the test file in a seperate repo instead though..